kb/2023-07-06/package_your_own_toolbox_image.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Package Your Own Toolbox Image
+description: How to package your own toolbox
+slug: package_your_own_toolbox_image
+authors:
+  - name: Vicente Cheng
+    title: Senior Software Engineer
+    url: https://github.com/Vicente-Cheng
+    image_url: https://github.com/Vicente-Cheng.png
+tags: [debug, harvester, container]
+hide_table_of_contents: false
+---
+
+Harvester OS is the immutable OS. That means you cannot install any packages on it. Although we provide the way to install packages (https://docs.harvesterhci.io/dev/troubleshooting/os#how-can-i-install-packages-why-are-some-paths-read-only). But it is not recommended to install packages on Harvester OS.
+
+We still need to debug with the system sometimes, so here we provide how to package your toolbox image. You can install any packages on it and debug with it.
+
+Giving an example as follow. If I would like to analyze the storage performance issue, I can install `blktrace` on the toolbox image.
+
+
+## Create a Dockerfile
+
+```bash
+FROM opensuse/tumbleweed
```

Comment:
suggest leap 15.4, which is more close to SLE Micro 5.3's OS

Reply:
Sure, updated!